### ISSUE TITLE
Sign the release package? 🙏

### ISSUE THREAD

**Issue by varenc** [Jaennaet/pISSStream#10]
Gentle request that the developer sign and notarize the release. 

Don't want onerous MDM policies preventing users from running this critical piece of software! Signing is also a requirement to be a registered homebrew cask, so I can just add `brew install --cask pissstream` to my new laptop provisioning script.

Also feel 100% free to close this issue with or without prejudice.

**Comment by phranck** [comment]
Yap, it should be signed.

**Comment by Jaennaet** [comment]
It absolutely should have something other than an ad hoc signature.

Unfortunately I don't have a paid Apple dev account or any plans to get one, and on top of that I'm not sure if there'd be a way to do that "official" signing so that it doesn't leak my name. Well, short of founding a strawman company for the dev license I guess? "pISSStream International plc" registered on the Caymans does sound kind of tempting.

**Comment by varenc** [comment]
You're right that I believe the only way to sign+notarize it without revealing your name is to register some sort of company/entity. As I recall Apple intentionally what's the name of "sole developers" to be visible. I'm not sure how they verify the name.

Though also you might find a strawman corporate entity useful for other purposes!
